text/reporting_assurance.tex

```diff
@@ -124,10 +124,10 @@ \subsubsection{The Assurances Extrinsic}
   &\mathsf{X}_A \equiv \token{\$jam\_available}
 \end{align}
 
-A bit may only be set if the corresponding core has a report pending availability on it which has not timed out:
+A bit may only be set if the corresponding core has a report pending availability on it:
```

Comment:
So, according to this change, now a bit can be set even if the pending report has timed out. Thus , if we achieve an availability supermajority for a timed-out report (according to 11.17) it would still be included in **W**.

![screenshot-2024-12-08-11-26-15](https://github.com/user-attachments/assets/50a25402-f20c-4b35-bb5c-172b2f665879)

This means it will still be considered for accumulation in the next pipeline stage.

Essentially, we're now a bit more lenient with timeouts. In other words, if it's timed out but still available, we no longer consider it an issue.

Given this explicit change, I assume this is an intentional design choice. However, I just wanted to double-check to be sure

Reply:
Yes - if a report is made available on the exact timeout block, we prefer to accumulate than to drop.